### ISSUE TITLE
Forge BE question - switching between models

### ISSUE THREAD

**Issue by alexdisablo** [lllyasviel/stable-diffusion-webui-forge#1031]
Hi all. I'm not sure this is an issue, but the thing I've noticed is that loading of models (switching between them) is what's taking most of the time for me during image generation. I'm using the same model (checkpoint), same ClipSkip. But this model are loaded all the time, multiple times during generation, each generation. 
Is there a way to avoid that ? I mean I don't change models, why system is loading them each time from the scratch ? wtf ?

Example of 1 banch of model swtiching (I can have like 10+ of them during 1 image generation)

```
Moving model(s) has taken 4.22 seconds
To load target model SDXL
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) =  6925.8046875
[Memory Management] Model Memory (MB) =  4897.086494445801
[Memory Management] Minimal Inference Memory (MB) =  1024.0
[Memory Management] Estimated Remaining GPU Memory (MB) =  1004.7181930541992
Moving model(s) has taken 22.20 seconds
100%|████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:06<00:00,  1.36it/s]
To load target model AutoencoderKL
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) =  6951.0546875
[Memory Management] Model Memory (MB) =  159.55708122253418
[Memory Management] Minimal Inference Memory (MB) =  1024.0
[Memory Management] Estimated Remaining GPU Memory (MB) =  5767.497606277466
Moving model(s) has taken 0.73 seconds
To load target model SDXLClipModel
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) =  6789.02978515625
[Memory Management] Model Memory (MB) =  2144.3546981811523
[Memory Management] Minimal Inference Memory (MB) =  1024.0
[Memory Management] Estimated Remaining GPU Memory (MB) =  3620.6750869750977
Moving model(s) has taken 5.66 seconds
To load target model SDXL
Begin to load 1 model
[Memory Management] Current Free GPU Memory (MB) =  6612.7646484375
[Memory Management] Model Memory (MB) =  4897.086494445801
[Memory Management] Minimal Inference Memory (MB) =  1024.0
[Memory Management] Estimated Remaining GPU Memory (MB) =  691.6781539916992
Moving model(s) has taken 31.47 seconds
100%|████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:07<00:00,  1.26it/s]
To load target model AutoencoderKL
```

**Comment by dongxiat** [comment]
yep, me too...this happen after [change Model Loading Behavior](https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/964). I think its move out when finish and move in again when click Generate

**Comment by alexdisablo** [comment]
> yep, me too...this happen after [change Model Loading Behavior](https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/964). I think its move out when finish and move in again when click Generate

The thing is that it's loading on and off not just 1 time during each generation. I press generate and it constantly changing models (like a lot of times, not just 1 initial load).

**Comment by alexdisablo** [comment]
@lllyasviel Is there a way to turn Model loading behavior off ? 
I'm working with the same checkpoint during the session, this feature kills the productivity dramatically. I understand it's useful when you work with multiple checkpoints, but I use the same one. And it keeps loading it from the scratch each generation, multiple times.

**Comment by andy8992** [comment]
I never use Git but honestly.... this is driving me insane, any speed gain I get by using distillation methods, 4 steps, or really any time saving method has been pretty much negated by the constant model unloading and loading.

I think I saw the announcement about model loading but this really really is a huge bummer for me. Currently comfy keeping a model loaded is a massive boon compared to this recently.

I switched to forget but I'm bummed that i may need to switch bad if it's going to constantly take this long to render even if I just rendered a second ago, hell this even happens during a batch of multiple image.s

Please give us the option to keep models loaded, this is pretty important to e.
